Tests/NIOTests/BootstrapTest.swift

```diff
@@ -529,6 +529,23 @@ class BootstrapTest: XCTestCase {
         XCTAssertNil(NIOPipeBootstrap(validatingGroup: elg))
         XCTAssertNil(NIOPipeBootstrap(validatingGroup: el))
     }
+    
+    func testShorthandOptionsAreEquivalent() throws {
+        func bindAndGetReuseAddrOption(applyBootstrapOptions : (ServerBootstrap) -> ServerBootstrap) throws -> ChannelOptions.Types.SocketOption.Value {
```

Comment:
generally, trailing closures shouldn't use argument labels. If you have them, it makes switching from the trailing closure to regular syntax hard:

### With label:

```
foo {
  bar()
}
```

vs

```
foo(label: {
    bar()
}
```

### Without label:

```
foo {
  bar()
}
```

vs

```
foo({
    bar()
}
```

so switching syntaxes is just adding/removing parens, no need to guess the label name.

Reply:
Done.